kibana/README.md

```diff
@@ -996,3 +996,64 @@ readonlyrest:
 ```
 
 Now try to login as user1, and ReadonlyREST Enterprise should initialise the index ".kibana\_user1" with all the index patterns and dashboards contained in the template tenancy.
+
+## Impersonation
+
+User with specific roles (`admin` or `unrestricted`) and permissions (`impersonation` section in the `readonlyrest.yml` file)
+can impersonate (imitate behaviors or actions of the user) another user.
+
+### Test settings
+
+During user impersonation, the `reasonlyrest.yml` settings are replaced by test settings and used during a whole impersonation process
+You need to configure test settings to be able to impersonate the user
+
+
+#### Setting test settings
```

Comment:
shouldn't we explain why?

This is important that the main (production) ROR engine is unaffected. It means that a user/admin is able to modify the test settings without worrying that their actions would break sth. 

But I'm not sure we should use vocabulary like "ROR engine". Any ideas on how to call it?

Reply:
Ok, What about:

`For impersonation to work, some valid test settings should be created and saved. It's important that the main ROR setting will be unaffected, so you, as an admin/user don't need to worry that you will break something. Here is how to write  test settings:`
?

Reply:
for me it's ok